lib/android-verify.js

```diff
@@ -44,118 +44,122 @@ let verifyAuthenticityV2 = (() => {
 
 function _asyncToGenerator(fn) { return function () { var gen = fn.apply(this, arguments); return new Promise(function (resolve, reject) { function step(key, arg) { try { var info = gen[key](arg); var value = info.value; } catch (error) { reject(error); return; } if (info.done) { resolve(value); } else { return Promise.resolve(value).then(function (value) { step("next", value); }, function (err) { step("throw", err); }); } } return step("next"); }); }; }
 
-var cbor = require('cbor');
-var URLSafeBase64 = require('urlsafe-base64');
-var r = require('jsrsasign');
-var request = require('isomorphic-fetch');
-var asn = require('asn1.js');
+const cbor = require('cbor');
+const URLSafeBase64 = require('urlsafe-base64');
+const r = require('jsrsasign');
+const request = require('isomorphic-fetch');
+const asn = require('asn1.js');
 const jsonSettings = require('../settings/settings.json');
 const R = require('ramda');
 // $FlowFixMe
 const Buffer = require('buffer').Buffer;
 
 const verify = exports.verify = (() => {
   var _ref = _asyncToGenerator(function* (data, version) {
```

Comment:
leftover var?

Reply:
yes!

Reply:
Oh wait, that is lib. Just review the src please